jena-rdfconnection/src/main/java/org/apache/jena/rdflink/RDFLinkHTTPBuilder.java

```diff
@@ -91,6 +91,7 @@ protected RDFLinkHTTPBuilder(RDFLinkHTTP base) {
         parseCheckUpdates   = base.parseCheckUpdates;
 
         querySendMode       = base.querySendMode;
+        updateSendMode      = base.updateSendMode;
```

Comment:
I noticed that this line was forgotten in https://github.com/apache/jena/pull/2253/files#diff-bee294b1706b3bd7768eb35b8cbcbb286197873cd3421a82602e4daf15f90dfaR93